recipes/ipopt/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+THIRDPARTY=(Metis Mumps)
```

Comment:
Be aware that the version of Metis downloaded here has a strange license.

You'll need `./get.ASL` too if you want an executable.